src/App.tsx

```diff
@@ -15,47 +16,111 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  if (sortType === SortType.ALPHABET) {
```

Comment:
This is 100% fine, although I would personally use switch - it works great with enum
`switch (sortType) {
    case NONE:
      break;
    case ALPHABET:
      visibleGoods.sort...` etc

Reply:
Agree, in this case it would look cleaner, more structured. Simply more readable ;)